### DISCUSSION TITLE
Virtual Select z-index problem

### DISCUSSION THREAD

**Discussion by yasinkt** [sa-si-dev/virtual-select#234] [Q&A] [answered]
Hello,

![image](https://user-images.githubusercontent.com/76478814/221363555-207a4150-8276-4de8-8523-86339cfa29a0.png)
My virtual select ⬆️

![image](https://user-images.githubusercontent.com/76478814/221363597-1448638e-1b40-447d-8e47-9512babc9287.png)
This is how it opens when I click on virtual select ⬆️

Why?

**Comment by sa-si-dev** [comment, marked as answer]
@yasinkt By default, dropbox would be partially visible if it is inside a overflow container.
To fix this issue use `dropboxWrapper: 'body'` property.
